### ISSUE TITLE
ydk doesn't seem to understand the "<" or ">" operations in the yang. 

### ISSUE THREAD

**Issue by manjuv1** [CiscoDevNet/ydk-gen#683]
Our yang model has a condition like below (sample)
```yang
   container timers {
      leaf config-1 {
        type uint16;
        default "60";
        units "seconds";
      }
      leaf config-2 {
        type uint16;
        default "180";
        units "seconds";
      }
      must "number(config-1) < number(config-2)" {
        error-message "config-1 should be lesser than config-2.";
        description
          "config-1 should be lesser than config-2";
      }
    }
```
But the ydk (version 0.6.2) doesn't seem to understand the "<" or ">" operations in the yang.

We get the below error on doing any crud operation on this model.
```
ydk.errors.YPYModelError:  Unexpected character(s) '&' (&lt; number(con). Path:
```

On analysing, we saw that YDK copies the yang model to ~/.ydk/<MachineIP:SessionID> after replacing all “<” to “&lt” and “>” to “&gt” and uses it for its CRUD operations. 

Is there any way to get past this error?

**Comment by ygorelik** [comment]
The issue related to presence of XML special character sequences like '<', '>', '&', and '"' has been resolved as part of #614.

**Comment by asteinhagen** [comment]
Hello Yan, Abhi,

hereby I verify that with the latest YDK bug #683 has been successfully fixed.

Thank you
